naked-objects/README.md

```diff
@@ -28,6 +28,10 @@ Use the Naked Objects pattern when
 
 * [Apache Isis](https://isis.apache.org/)
 
+## Reference
+
+* [Apache Isis SimpleApp](https://github.com/apache/isis-app-simpleapp)
+
```

Comment:
Please add a section for an explanation that describes how the pattern works in practice. See the other patterns for examples of how it has been implemented

Additionally, it might be good to point out that `README` contains instructions on how to run this example.